troposphere/static/js/globals.js

```diff
@@ -2,21 +2,23 @@ import jstz from 'jstz';
 
 
 let timezone = jstz.determine();
-let tz_region = timezone ? timezone.name() : 'America/Phoenix';
-let shell_proxy = 'https://atmo-proxy.cyverse.org/';
+let tz_region = timezone ? timezone.name() : "America/Phoenix";
+let shell_proxy = "https://atmo-proxy.cyverse.org/";
+let default_footer_link = "http://www.cyverse.org/";
```

Comment:
Forward thought: maybe we should make our defaults cyverse agnostic.

In the near future when it's super easy to create a new Atmosphere, orgs trying out our service will think its easy to configure if it doesn't have these traces of the original.